azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -1369,19 +1363,17 @@ public String handle(ResultSet rs) throws SQLException {
     // Select running and executor assigned flows
     private static String FETCH_ACTIVE_EXECUTABLE_FLOW =
       "SELECT ex.exec_id exec_id, ex.enc_type enc_type, ex.flow_data flow_data, et.host host, "
-        + "et.port port, ax.update_time axUpdateTime, et.id executorId, et.active executorStatus"
+        + "et.port port, et.id executorId, et.active executorStatus"
         + " FROM execution_flows ex"
         + " INNER JOIN "
-        + " active_executing_flows ax ON ex.exec_id = ax.exec_id"
-        + " INNER JOIN "
-        + " executors et ON ex.executor_id = et.id";
+        + " executors et ON ex.executor_id = et.id"
+        + " Where ex.status NOT IN (?, ?, ?)";
```

Comment:
Because currently there are 13 statuses in total. 3 of them are finished statuses. So it's more concise in the query to just exclude the finished statuses.